p5-travel-app/src/client/index.js

```diff
@@ -35,6 +36,7 @@ const handleSearch = async (e) => {
   trip.country = countryInfo.name;
   trip.countryFlag = countryInfo.flag;
   trip.image = await getImageURL(trip.city, trip.country);
+  console.log(trip);
```

Comment:
remove

Reply:
done, clean it in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)